src/sentry/data_export/tasks.py

```diff
@@ -52,6 +52,8 @@ def assemble_download(
     offset=0,
     bytes_written=0,
     environment_id=None,
+    retries=3,
```

Comment:
Could we rename this to
```suggestion
    export_retries=3,
```
since this is the retries for the entire export rather than for the individual task?

Reply:
renamed in 8ee91de